packages/react/src/Banner/Banner.module.css

```
@@ -0,0 +1,194 @@
+    --banner-icon-fgColor: var(--fgColor-accent);
+  }
+
+  &[data-variant='success'] {
+    --banner-bgColor: var(--bgColor-success-muted);
+    --banner-borderColor: var(--borderColor-success-muted);
+    --banner-icon-fgColor: var(--fgColor-success);
+  }
+
+  &[data-variant='upsell'] {
+    --banner-bgColor: var(--bgColor-upsell-muted);
+    --banner-borderColor: var(--borderColor-upsell-muted);
+    --banner-icon-fgColor: var(--fgColor-upsell);
+  }
+
+  &[data-variant='warning'] {
+    --banner-bgColor: var(--bgColor-attention-muted);
+    --banner-borderColor: var(--borderColor-attention-muted);
+    --banner-icon-fgColor: var(--fgColor-attention);
+  }
+}
+
+/* BannerContainer -------------------------------------------------------- */
+
+.BannerContainer {
+  font-size: var(--text-body-size-medium, 0.875rem);
+  align-items: start;
+  line-height: var(--text-body-lineHeight-medium, calc(20 / 14));
+  row-gap: var(--base-size-4, 0.25rem);
+  column-gap: var(--base-size-4, 0.25rem);
+}
+
+.Banner :where(.BannerContainer) {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.Banner[data-dismissible] .BannerContainer {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto;
+}
+
+/* BannerContent ---------------------------------------------------------- */
+
+.BannerContent {
+  display: grid;
+  row-gap: var(--base-size-4, 0.25rem);
+  grid-column-start: 1;
+  margin-block: var(--base-size-8, var(--base-size-8));
+}
+
+.Banner[data-title-hidden] .BannerContent {
+  margin-block: var(--base-size-6, var(--base-size-6));
+}
+
+@media screen and (min-width: 544px) {
+  .BannerContent {
+    flex: 1 1 0%;
+  }
+}
+
+.BannerTitle {
+  margin: 0;
+  font-size: inherit;
+  font-weight: var(--base-text-weight-semibold, 600);
+}
+
+/* BannerIcon ------------------------------------------------------------- */
+
+.BannerIcon {
+  display: grid;
+  place-items: center;
+  padding: var(--base-size-8, var(--base-size-8));
+}
+
+.BannerIcon svg {
+  /* 20px is the line box height of the trailing action buttons */
+  height: var(--base-size-20, 1.25rem);
+  color: var(--banner-icon-fgColor);
+  fill: var(--banner-icon-fgColor);
+}
+
+.Banner[data-title-hidden] .BannerIcon svg {
+  height: var(--base-size-16, 1rem);
+}
+
+/* BannerDismiss ---------------------------------------------------------- */
+
+.BannerDismiss {
+  display: grid;
+  place-items: center;
+  padding: var(--base-size-8, var(--base-size-8));
+  margin-inline-start: var(--base-size-4, var(--base-size-4));
+}
+
+.BannerDismiss svg {
+  color: var(--banner-icon-fgColor);
+}
+
+/* BannerActions ---------------------------------------------------------- */
+
+.BannerActionsContainer {
+  display: flex;
+  column-gap: var(--base-size-12, 0.5rem);
+  align-items: center;
+}
+
+.BannerActions :where([data-primary-action='trailing']) {
+  display: none;
+}
+
+@media screen and (min-width: 544px) {
```

```suggestion
@media screen and (--viewportRange-regular) {
```

This custom media query should be the same here, 🤞🏻